src/javascript/app/base/client.js

```diff
@@ -89,14 +99,14 @@ const Client = (() => {
         ClientBase.set('loginid', '');
         SocketCache.clear();
         RealityCheckData.clear();
-        LiveChat.endLiveChat().then(() => {
-            const redirect_to = getPropertyValue(response, ['echo_req', 'passthrough', 'redirect_to']);
-            if (redirect_to) {
-                window.location.href = redirect_to;
-            } else {
-                window.location.reload();
-            }
-        });
+        if (window.location.hostname === ('staging.binary.com' || 'binary.com')){
```

Comment:
```suggestion
const url = new URL(window.location.href);
const domain = url.hostname.split('.').slice(-2).join('.');
        if (domain === 'binary.com') {
```
This way u only need to check for the `root` domain and no need to check explicitly for the subdomains